src/components/ClusterOverview/Details/storageDetails.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { get } from 'lodash';
+
+import { ClusterOverviewContextGenericConsumer } from '../ClusterOverviewContext';
+import { InlineLoading } from '../../Loading';
+import { Details } from './Details';
+
+export const StorageDetails = ({LoadingComponent }) => {
+  const items = {
+    name: {
+      title: 'Name',
+      value: 'Cluster 1',
+      isLoading: false,
+    },
+    provider: {
+      title: 'Provider',
+      //value: get(infrastructure, 'status.platform'),
+      value: 'Ceph',
```

Comment:
is this going to be hardcoded for the demo too ?

Reply:
Yes, it will be hardcoded.